core/node/proof_data_handler/src/tee_request_processor.rs

```diff
@@ -208,6 +204,28 @@ impl TeeRequestProcessor {
         )
         .await?;
 
+        let sealed_at = connection
+            .blocks_dal()
+            .get_sealed_at(l1_batch_number)
+            .await?;
+
+        let duration =
+            sealed_at.and_then(|sealed_at| (Utc::now() - sealed_at.and_utc()).to_std().ok());
+
+        let duration_secs_f64 = if let Some(duration) = duration {
+            METRICS.tee_proof_roundtrip_time[&proof.0.tee_type.into()].observe(duration);
+            duration.as_secs_f64()
+        } else {
+            f64::NAN
+        };
+
+        tracing::info!(
+            l1_batch_number = %l1_batch_number,
+            sealed_to_proven_in_secs = %duration_secs_f64,
```

Comment:
Nit: `f64` values can be supplied to `tracing` macros directly, w/o a `Display` / `%` qualifier.

Reply:
done